api/src/main/java/org/openmrs/api/CohortService.java

```diff
@@ -143,21 +143,6 @@
 	 */
 	public List<Cohort> getCohorts(String nameFragment) throws APIException;
 	
-	/**
-	 * @deprecated use {@link #getCohortsContainingPatientId(Integer)}
-	 *
-	 * Find all Cohorts that contain the given patient. (Not including voided Cohorts)
-	 * 
-	 * @param patient patient used to find the cohorts
-	 * @return All non-voided Cohorts that contain the given patient
-	 * @throws APIException
-	 * @should not return voided cohorts
-	 * @should return cohorts that have given patient
-	 */
-	@Deprecated
-	@Authorized({ PrivilegeConstants.GET_PATIENT_COHORTS })
```

Comment:
We do not deprecate and remove at the same time. After deprecation, we still leave it in for a number of versions to come until when we eventually remove it.

Reply:
@dkayiwa Fixed